Fix font loading in storybook

b12f opened the pull request:
Both Lato and the icon font were not loading correctly since the CSS expects them to be available under a certain URL. This URL is available in the core app, but not storybook. This commit makes sure the assets load correctly in Storybook, giving us pretty icons and the right font.

Commit:
Fix font loading in storybook

Both Lato and the icon font were not loading correctly since the CSS
expects them to be available under a certain URL. This URL is available
in the core app, but not storybook. This commit makes sure the assets
load correctly in Storybook, giving us pretty icons and the right font.

psatyal commented:
Looking forward to reviewing this! There seems to be a problem with the pullpreview however; it says "succesfully deployed" but there's no link, and the action had an error associated with it.

I've removed and added the label again, let's see...

**Update, 21h39:** Again, error + "successfully deployed" + no link to deployment :/

<img width="959" alt="Screenshot 2022-10-31 at 21 38 22" src="https://user-images.githubusercontent.com/88370597/199105906-f43f17b7-a606-4922-9aec-d043cabf631d.png">

Kharonus commented:
I checked that branch out locally, and there it works:
![image](https://user-images.githubusercontent.com/38206611/199246598-e619bfa7-6848-4d6e-b5cd-03996cf4096c.png)

How can we fix the pullpreview?

b12f commented:
It's this issue: https://github.com/storybookjs/storybook/issues/18686

I'll see if we can work around it.

Commit:
Change asset loading for storybook

Storybook assets cannot currently be loaded by merging directories,
so this fix makes sure files are loaded individually.

https://github.com/storybookjs/storybook/issues/18686

Kharonus reviewed (approved):
LGTM
